operations/image/dffml_operations_image/operations.py

```diff
@@ -0,0 +1,42 @@
+from typing import List
+import cv2
+import numpy as np
+
+from dffml.df.base import op
+
+from .definitions import (
+    resized_array,
+    data_array,
+    old_dimensions,
+    new_dimensions,
+)
+
+
+@op(
+    inputs={
+        "data": data_array,
+        "old_dim": old_dimensions,
+        "new_dim": new_dimensions,
+    },
+    outputs={"resized_array": resized_array},
+    conditions=[],
+)
+async def resize(data: np.ndarray, old_dim: List[int], new_dim: List[int]):
```

Comment:
We can probably use the auto definition creation here

```suggestion
async def resize(data: List[int], old_dim: List[int], new_dim: List[int]) -> Tuple[int]:
```

Reply:
Here, I am thinking to modify the configloader to return a numpy array after .flatten() and not change it to tuple or list. Please refer to the second paragraph in this comment https://github.com/intel/dffml/pull/708#issuecomment-643649296

What do you think?

Reply:
It's okay if feature data in a Record is a numpy type. I made this sepcific comment because you immediately change `data` into a `np.array`.

```python
np.array(data).reshape(old_dim)
```

Reply:
Unless there's a compelling reason to not use the `op` auto definition stuff, we should probably use it here to decrease number of lines.